src/collects/seashell/backend/project.rkt

```diff
@@ -440,6 +488,22 @@
     [else
       (values #f `#hash((messages . ,messages) (status . "compile-failed")))]))
 
+
+;; (compile-and-run-project/use-runner name tests)
+;; is a wrapper around compile-and-run-project, supplying the file
+;; from the project settings file.
+;; Note: the file arg is used only to determine the question
+(define/contract (compile-and-run-project/use-runner name file tests)
```

Comment:
I would prefer if we got the "question" directory here instead of breaking up the file path to find the question directory.